src/utils/hooks.tsx

```diff
@@ -338,7 +357,7 @@ export function useForumData(
             posts,
             restriction,
             moderatorMint,
-            votes
+            votes: onChainAccountNotFound(),
```

Comment:
I think it makes more sense to set this one to `initial()` rather than not found, because it's going to be loaded, and we don't know conclusively that it doesn't exist

Reply:
it seems the types here are incompatible? so I can just add the `Initial` type to `LoadingResult`